packages/manager/modules/product-offers/src/product-offers.html

```diff
@@ -65,6 +65,15 @@
         ></p>
     </oui-step-form>
 
+    <button
+        type="button"
+        class="oui-button oui-button_l oui-button_primary"
+        data-ng-click="$ctrl.workflow.validateOffer()"
+        data-ng-if="$ctrl.workflowOptions.expressOrder && $ctrl.workflow.pricings.length === 1"
+    >
+        <span data-translate="ovh_product_offers_order_express_submit"></span>
+    </button>
```

Comment:
I think we don't need create new button but we can change the rule for don't automaticaly select the price if we have only one price in case where we are in expressOrder configuration

if (this.hasUniquePricing() && !this.expressOrder) {
in class order / service